x12-edi/2.5/modules/ROOT/pages/x12-edi-connector-hipaa.adoc

```diff
@@ -1,23 +1,26 @@
-= X12 EDI Connector HIPAA 2.5 - Mule 4
+= X12 EDI Connector 2.5 - HIPAA - Mule 4
 :page-aliases: connectors::x12-edi/x12-edi-connector-hipaa.adoc
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Premium]
 
-Anypoint Connector for X12 EDI Connector HIPAA provides support for the HIPAA (Health Insurance Portability and Accountability Act) standards based on the X12 standards and version 005010.
-This information provides details about using the X12 Module for HIPAA transaction sets, and how usage differs from the standard X12.
-For complete information on X12 standards, refer to xref:index.adoc[X12 EDI Connector 2.5] and xref:x12-edi-schema-language-reference.adoc[X12 EDI Schema Language Reference].
+Anypoint Connector for X12 EDI Connector 2.5 - HIPAA provides support for the Health Insurance Portability and Accountability Act (HIPAA) standards based on the X12 Connector standards and version `005010`.
+Refer to this information for using the X12 Connector for HIPAA transaction sets, and for how usage differs from the standard X12.
+For complete information on X12 standards, refer to xref:index.adoc[X12 EDI Connector 2.5] and xref:x12-edi-schema-language-reference.adoc[EDI Schema Language Reference].
 
-The X12 Module HIPAA support includes:
+The X12 Connector - HIPAA support includes:
+
+* X12 HIPAA EDI message reading, message validation, and message writing
 
-* X12 HIPAA EDI message-reading, message-validation, and message-writing
 * Integration with DataSense and DataWeave
-* HIPAA message packs for 005010X212, 005010X214B3, 005010X217, 005010X218, 005010X220A1, 005010X221A1, 005010X222A1, 005010X222A2, 005010X223A2, 005010X223A3, 005010X224A2, 005010X224A3, and 005010X279A1
+
 * The ability to customize the base X12 HIPAA schemas
 
-Most aspects of using HIPAA with the X12 Module are the same as using standard X12. The main differences are:
+Most aspects of using HIPAA with the X12 Connector are the same as using standard X12. The main differences are:
```

Comment:
"with X12 Connector" (Kim always has us remove "the."

Reply:
Fixed throughout.